clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -768,4 +770,11 @@ public ListConsumerGroupOffsetsResult listConsumerGroupOffsets(String groupId) {
     public DeleteConsumerGroupsResult deleteConsumerGroups(Collection<String> groupIds) {
         return deleteConsumerGroups(groupIds, new DeleteConsumerGroupsOptions());
     }
+
+    /**
+     * Get the metrics kept by the adminClient
+     *
+     * @return
+     */
+    public abstract Map<MetricName, ? extends Metric> metrics();
```

Comment:
Since this is a public class we'd need a KIP for this change. But it should be straight-forward (we should probably do this since day one). WDYT @cmccabe @hachikuji